src/graphql/admin/root/query/account-details-by-phone.ts

```diff
@@ -0,0 +1,27 @@
+import { GT } from "@graphql/index"
+
+import GraphQLUser from "@graphql/admin/types/object/account"
+import Phone from "@graphql/types/scalar/phone"
+
+import { getAccountByUserPhone } from "@app/admin"
+
+const AccountDetailsByUserPhoneQuery = GT.Field({
+  type: GT.NonNull(GraphQLUser),
+  args: {
+    phone: { type: GT.NonNull(Phone) },
+  },
+  resolve: async (parent, { phone }) => {
+    if (phone instanceof Error) {
+      throw phone
```

Comment:
I saw this was from a simple switchover, but should this be `return` instead of `throw` here?

Reply:
every `error` in graphql should throw no?

@samerbuna @bodymindarts

Reply:
that was true until the new error map was introduced (this applies for V2, for v1 is not necessary)... now should be something like:

for graphql input validations (line 14 repace input by phone) 
```
if (input instanceof Error) {
  return { errors: [{ message: input.message }] }
}
```
and for returns from app layer (line 19 replace result with account)
```
if (result instanceof Error) {
  const appErr = mapError(result)
  return { errors: [{ message: appErr.message }] }
}
```

Reply:
since there are many mutations/queries missing mapError I think is better to do it in another PR